.changeset/six-keys-cheat.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
+---
+
+Remove legacy handling for MDX content collections. Ensure you are using @astrojs/mdx v0.18 and above.
```

Comment:
```suggestion
Remove legacy handling for MDX content collections. Ensure you are using `@astrojs/mdx` v0.18 or above.
```
Great catch, @ematipico! Would also point out that you can't use ALL the versions including and above v0.18 at once... :wink: